src/content/components/modal/usage.mdx

```diff
@@ -3,32 +3,43 @@ title: Modal
 tabs: ['Code', 'Usage', 'Style']
 ---
 
+### Modals communicate information via a secondary window and allow the user to maintain the context of a particular task.
+
 <AnchorLinks>
 
 - [General guidance](#general-guidance)
 - [Format](#format) 
 - [Variations](#variations) 
-- [Dismissal](#dismissal)
 
 </AnchorLinks>
 
 ## General guidance
 
-_Modals_ communicate information via a secondary window and allow the user to maintain the context of a particular task. Use modals sparingly because they interrupt user workflow.
+Modals interrupt user workflow by design. They are most effective when a task must be completed before a user can continue. While effective when used correctly, modals should be used sparingly to limit disruption to a user experience.
+
+#### Dismissal
+
+Modals may be dismissed in 3 ways:
+
+- Using the “✕” in the upper right-hand corner of the modal
+- Pressing the `ESC` key
+- Clicking or touching outside of the modal
 
 ## Format
 
+The modal is composed of three distinct zones: A header, the body, and a footer. Components (eg. data table, form, progress indicator) should occupy the full width of the modal. Text, including the paragraph component, should only be 75% of the modal's width.
```

Comment:
> Components (eg. data table, form, progress indicator) **should** occupy the full width of the modal. Text, including the paragraph component, should only be 75% of the modal's width.

I will change that to 

> Components (eg. data table, form, progress indicator) **may** occupy up to full width of the modal. Text, including the paragraph component, should only use 75% of the modal's width.

Reply:
Also, feels like the width guidance potentially belong to **Body** section? It doesn't apply to header and footer.

Reply:
Changes made!